contracts/RMRK/nesting/RMRKNesting.sol

```diff
@@ -671,34 +793,36 @@ contract RMRKNesting is Context, IERC165, IERC721, IRMRKNesting, RMRKCore {
     }
 
     /**
+     * @notice Used to enforce that the given token has been minted.
      * @dev Reverts if the `tokenId` has not been minted yet.
+     * @dev The validation checks whether the owner of a given token is a `0x0` address and considers it not minted if
+     *  it is. This means that both tokens that haven't been minted yet as well as the ones that have already been
+     *  burned will cause the transaction to be reverted.
+     * @param tokenId ID of the token to check
      */
-
     function _requireMinted(uint256 tokenId) internal view virtual {
         if (!_exists(tokenId)) revert ERC721InvalidTokenId();
     }
 
     /**
-     * @dev Returns whether `tokenId` exists.
-     *
-     * Tokens can be managed by their owner or approved accounts via {approve} or {setApprovalForAll}.
-     *
-     * Tokens start existing when they are minted (`_mint`),
-     * and stop existing when they are burned (`burn`).
+     * @notice Used to check whether the given token exists.
+     * @dev Tokens can be managed by their owner or approved accounts via {approve} or {setApprovalForAll}.
```

Comment:
This comment feels out of place here.

Reply:
Yeah, this seemed to have snuck in totally randomly :'D